src/components/hooks/useMeeting.js

```diff
@@ -49,7 +51,7 @@ export default function useMeeting(meetingID) {
       throw error;
     };
     const onComplete = () => {
-      setMeeting(emptyMeeting);
+      setMeeting({...emptyMeeting, state: MeetingState.LEFT});
```

Comment:
the `LEFT` state is already set by the adapter here https://github.com/webex/components/pull/201/files?file-filters%5B%5D=.js#diff-b63aa3e7e2482c57f9d72c2a873dafae6bdaedaa8c04639c9313b922df39ebd9R273

we don't need to do anything in this `onComplete` handler, it can be removed.